schedule/yast/raid/raid1_sle_lvm.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           lvm+raid1@64bit
+description:    >
+  Validation of partitioning for raid1 on lvm
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+  LVM: 1
+test_data:
+  !include: schedule/yast/raid/testdata/lvm_raid1.yaml
+schedule:
+  - installation/isosize
+  - installation/bootloader
```

Comment:
Use `bootloader_start` like in here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/yast/raid/raid0_sle_gpt_uefi.yaml#L9 so you don't need a different schedule for aarch: `schedule/yast/raid/raid1_sle_aarch64_lvm.yaml`